panel/io/resources.py

```diff
@@ -183,7 +183,11 @@ def loading_css():
     }}
     """
 
-def patch_stylesheet(stylesheet, dist_url):
+def patch_stylesheet(stylesheet, dist_url) -> None:
+    global RESOURCE_MODE
+    if RESOURCE_MODE == "server":
+        return
```

Comment:
After merging latest changes from main this should not be needed.

Reply:
You are right. I have reverted the change.